### PR TITLE
mep12 on scatter_profile.py

### PR DESCRIPTION
**Pull request by ericmjl**

### CODE REVIEW

File: examples/pylab_examples/scatter_profile.py [hunk 1]
```diff
@@ -9,16 +6,16 @@
...
 import time
 
-
 for N in (20, 100, 1000, 10000, 50000):
     tstart = time.time()
-    x = 0.9*pylab.rand(N)
-    y = 0.9*pylab.rand(N)
-    s = 20*pylab.rand(N)
-    pylab.scatter(x, y, s)
+    x = 0.9*np.rand(N)
```
Comment (jenshnielsen) [inline examples/pylab_examples/scatter_profile.py]: should be np.random.rand

File: examples/pylab_examples/scatter_profile.py [hunk 2]
```diff
@@ -9,16 +6,16 @@
...
 import time
 
-
 for N in (20, 100, 1000, 10000, 50000):
     tstart = time.time()
-    x = 0.9*pylab.rand(N)
-    y = 0.9*pylab.rand(N)
-    s = 20*pylab.rand(N)
-    pylab.scatter(x, y, s)
+    x = 0.9*np.random.random(N)
```
Comment (jenshnielsen) [inline examples/pylab_examples/scatter_profile.py]: Sorry it has to be np.random.rand

Comment (ericmjl) [inline examples/pylab_examples/scatter_profile.py]: Ack! I will get that changed as soon as I get back to my computer!



Cheers,Eric

**Comment by jenshnielsen** [comment]
Thanks @ericmjl Do you have the time to fix #4646 and #4631 too?

**Comment by ericmjl** [comment]
Just did it! :smile:

**Pull request merged**